planner/core/common_plans.go

```diff
@@ -471,6 +487,32 @@ func (e *Execute) rebuildRange(p Plan) error {
 			}
 		}
 	case *PointGetPlan:
+		// if access condition is not nil, which means it's a point get generated by cbo.
+		if x.AccessConditions != nil {
```

Comment:
Why do we need to add this logic?  are `Handle` / `HandleParam` and `IndexValues` / `IndexValueParams` not enough to cover clustered index cases?

Reply:
Because for point-get in CBO, it doesn't have `HandleParam` or `IndexValueParams`.